specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/definitions.json

```diff
@@ -7986,6 +7986,109 @@
       },
       "description": "Properties of the Gateway contract."
     },
+    "GatewayConfigConnectionBaseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current provisioning state of the API Management gateway config connection which can be one of the following: Created/Activating/Succeeded/Updating/Failed/Stopped/Terminating/TerminationFailed/Deleted.",
+          "enum": [
+            "Created",
+            "Activating",
+            "Succeeded",
+            "Updating",
+            "Failed",
+            "Stopped",
+            "Terminating",
+            "TerminationFailed",
+            "Deleted"
```

Comment:
>Deleted [](http://example.com/codeflow?start=13&length=7)

You normally shouldn't have a Deleted provisioning state. You should just be returning 404 if the resoruce is already deleted.
A `Deleting` state would make sense to me though, if you have long-running delete, lots of RPs do that.

Reply:
removed